external-docs/docs/glossary/index.md

```diff
@@ -1,3 +1,70 @@
 # Glossary
 
-todo: Write something here. Issue #259.
+### Aggregate
+An aggregate is formed when multiple numbers are gathered for statistical purposes and are expressed as one number. This could be in the form of a total or an average<sup>[[10]](https://ec.europa.eu/eurostat/statistics-explained/index.php?title=Beginners:Statistical_concept_-_Aggregate)</sup>.
+
+### Attribute
+The attribute components allow us to qualify and interpret the observed value(s). They 	enable specification of the units of measure, any scaling factors and metadata such as the 	status of the observation (e.g. estimated, provisional)<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
+
+### Code List
+Predefined set of terms from which some statistical coded concepts take their values<sup>[[4]](https://sdmx.org/wp-content/uploads/SDMX_Glossary_version_2_1-Final-2.docx)</sup>.\
```

Comment:
It'd be nice to make this definition simpler. I understand what you mean here but I suspect most new users coming to this will struggle to follow it.

Perhaps something like `Predefined set of codified concepts which represent the distinct values that a dimension can hold.`